spark/dl/src/main/scala/com/intel/analytics/bigdl/transform/vision/image/RoiMiniBatch.scala

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
I prefer to push this class into the same file as `ClassificationMTImageFeatureToBatch` to keep hierarchy consistency.